proto/primitives/astria/primitive/v1/types.proto

```diff
@@ -48,4 +48,6 @@ message RollupId {
 // using the first 20 bytes of the sha256 hash.
 message Address {
   bytes inner = 1;
+
+  string bech32m = 2;
```

Comment:
Can we add some comments in here on the use within the protos?

Reply:
Added comments on the fields.